moto/elasticache/models.py

```diff
@@ -334,6 +417,45 @@ def delete_cache_cluster(self, cache_cluster_id: str) -> CacheCluster:
                 return cache_cluster
         raise CacheClusterNotFound(cache_cluster_id)
 
+    def create_cache_subnet_group(
+        self,
+        cache_subnet_group_name: str,
+        cache_subnet_group_description: str,
+        subnet_ids: List[str],
+        tags: Optional[List[Dict[str, str]]],
+    ) -> CacheSubnetGroup:
+        if cache_subnet_group_name in self.cache_subnet_groups:
+            raise CacheSubnetGroupAlreadyExists(cache_subnet_group_name)
+
+        cache_subnet_group = CacheSubnetGroup(
+            account_id=self.account_id,
+            region_name=self.region_name,
+            cache_subnet_group_name=cache_subnet_group_name,
+            cache_subnet_group_description=cache_subnet_group_description,
+            subnet_ids=subnet_ids,
+            tags=tags,
+        )
+        self.cache_subnet_groups[cache_subnet_group_name] = cache_subnet_group
+        return cache_subnet_group
+
+    @paginate(PAGINATION_MODEL)
+    def describe_cache_subnet_groups(
+        self,
+        cache_subnet_group_name: str,
+        max_records: int,
+        marker: str,
+    ) -> List[CacheSubnetGroup]:
+        if max_records is None:
+            max_records = 100
+        if cache_subnet_group_name:
+            if cache_subnet_group_name in self.cache_subnet_groups:
+                cache_subnet_group = self.cache_subnet_groups[cache_subnet_group_name]
+                return list([cache_subnet_group])
+            else:
+                raise CacheSubnetGroupNotFound(cache_subnet_group_name)
+        cache_subnet_groups = list(self.cache_subnet_groups.values())[:max_records]
```

Comment:
```suggestion
    def describe_cache_subnet_groups(
        self,
        cache_subnet_group_name: str,
    ) -> List[CacheSubnetGroup]:
        if cache_subnet_group_name:
            if cache_subnet_group_name in self.cache_subnet_groups:
                cache_subnet_group = self.cache_subnet_groups[cache_subnet_group_name]
                return list([cache_subnet_group])
            else:
                raise CacheSubnetGroupNotFound(cache_subnet_group_name)
        cache_subnet_groups = list(self.cache_subnet_groups.values())
```
Second nitpick. Your approach worked - but all the pagination-details are already handled by the decorator, so it's not necessary to duplicate that logic